scss/titles.scss

```diff
@@ -0,0 +1,4 @@
+.title {
+	color: #000;
```

Comment:
You can move both this color and the size to constants, so they can be reused in the whole app 😁 
In order to do this, you may create colors.scss and sizes.scss files, that should contain only constants.

Reply:
@marcalmon recuerda en los archivos que te menciona Pablo agregar los valores que te menciona esta card: https://trello.com/c/BRwhE9Qo/11-setup-del-proyecto